net/subversion/Makefile

```diff
@@ -101,6 +102,9 @@ else
 	CONFIGURE_ARGS += --disable-nls
 endif
 
+CONFIGURE_VARS += \
+	LDFLAGS="$(TARGET_LDFLAGS) $(APU_LIBS)"
```

Comment:
I'd prefer simply "TARGET_LDFLAGS += $(APU_LIBS)"

Reply:
Indeed! Saves a few bytes and probably makes this Makefile a little more readable.